app/validators/questions/__init__.py

```diff
@@ -10,10 +10,10 @@
 from app.validators.questions.question_validator import QuestionValidator
 
 
-def get_question_validator(question):
+def get_question_validator(question, data_version):
     validators = {
         "Calculated": CalculatedQuestionValidator,
         "DateRange": DateRangeQuestionValidator,
         "MutuallyExclusive": MutuallyExclusiveQuestionValidator,
     }
-    return validators.get(question["type"], QuestionValidator)(question)
+    return validators.get(question["type"], QuestionValidator)(question, data_version)
```

Comment:
Having a look at the code base, it would be better to validate this in https://github.com/ONSdigital/eq-questionnaire-validator/blob/master/app/validators/answers/__init__.py#L8-L23

It already passes the schema in conditionally, you can just always pass it in meaning you can get data version.